modules/channel-web/src/views/lite/main.tsx

```diff
@@ -175,6 +176,11 @@ class Web extends React.Component<MainProps> {
       return
     }
 
+    if (Number(this.props.config.conversationId) !== Number(event.conversationId)) {
```

Comment:
Technically if you don't configure conversationId (which is the default setting), it will result in NaN !== 13, so it would never add to the conversation. You need a separate check to see IF it was configured before